virt/virt-4-12-release-notes.adoc

```diff
@@ -140,14 +140,17 @@ The SVVP Certification applies to:
 
 Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
-* In a future release, support for the legacy HPP custom resource, and the associated storage class, will be deprecated. Beginning in {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. The Operator continues to support the existing (legacy) format of the HPP custom resource and the associated storage class. If you use the HPP Operator, plan to xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[create a storage class for the CSI driver] as part of your migration strategy.
+ 
 
 
 [id="virt-4-12-removed"]
 === Removed features
 
 Removed features are not supported in the current release.
 
+// CNV-23499_412RN
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed. In {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage.
```

Comment:
Sorry for the delay. I had to clarify this with the storage development team stakeholders. Please make the following change:
Support for the legacy HPP custom resource, and the associated storage class, has been removed. -> Support for the legacy HPP custom resource, and the associated storage class, has been removed for all new deployments. 

In {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. -> [Add this] Only in the case of an upgrade will a previously installed legacy HPP custom resource still be supported.